models/levels.js

```diff
@@ -0,0 +1,60 @@
+const firestore = require("../utils/firestore");
+const levelModel = firestore.collection("levels");
+
+/**
+ *
+ * @param levelData { Object }: Data of the level
+ * @returns {Promise<tagModel|Object>}
+ */
+
+const addLevel = async (levelData) => {
+  try {
+    const { id } = await levelModel.add(levelData);
+    return { id, levelData };
+  } catch (err) {
+    logger.error("Error in creating Level", err);
+    throw err;
+  }
+};
+
+/**
+ *
+ * @param id { String }: id of the level to be deleted
+ * @returns {Promise<tagModel|String>}
+ */
+
+const deleteLevel = async (id) => {
+  try {
+    await levelModel.doc(id).delete();
```

Comment:
can we destructure `id` here because I saw that you are sending req.body in the argument there

Reply:
fixed it.

Reply:
Why do we need an id in return response?

We are deleting data, which means we don't require any more.
it should only success message ``` Data deleted successfully```

Reply:
makes sense let me do it